index.rst

```diff
@@ -1,61 +1,311 @@
+Some background and relevant information about the MTAOS can be found in the `AOS Workshop`_ confluence page.
 
-  and
+In addition to being an extremely critical component for operating the Main Telescope (MT) the MTAOS is also unique in the sense that it interfaces with all other sub-systems, e.g.;
 
-  Subsubsections
-  ^^^^^^^^^^^^^^
+  - Camera, to acquire the intra/extra or corner rafts data for wavefront error processing.
+  - Data Management, to process the data with instrument signature removal (ISR) and wavefront estimation pipeline (WEP).
+  - Telescope and Site, mainly to apply optical corrections.
 
-  To add images, add the image file (png, svg or jpeg preferred) to the
-  _static/ directory. The reST syntax for adding the image is
+Here we explore updates to the `MTAOS (xml) interface`_ and overall software architecture to align it with the evolution of the system design.
+The most significant advancement that affect how the MTAOS operates are;
 
-  .. figure:: /_static/filename.ext
-     :name: fig-label
+  - Better understanding of the observatory operation procedures.
+  - Consolidation of the data access interface, through butler gen3, for local data processing of corner wavefront sensors and remote access for full array mode.
+  - Consolidation of the OCS Controlled Pipeline System (`OCPS`_) for remote data processing.
+  - Conversion of the wavefront estimation pipeline (`WEP`_) into a butler gen3 pipeline task.
 
-     Caption text.
+.. _MTAOS: https://ts-mtaos.lsst.io/index.html
+.. _AOS Workshop: https://confluence.lsstcorp.org/x/zzTbAw
+.. _MTAOS (xml) interface: https://ts-xml.lsst.io/sal_interfaces/MTAOS.html
+.. _OCPS: https://dmtn-133.lsst.io
+.. _WEP: https://github.com/lsst-ts/ts_wep
 
-   Run: ``make html`` and ``open _build/html/index.html`` to preview your work.
-   See the README at https://github.com/lsst-sqre/lsst-technote-bootstrap or
-   this repo's README for more info.
+.. _Data-access-changes:
 
-   Feel free to delete this instructional comment.
+Data access changes
+===================
 
-:tocdepth: 1
+The consolidation of data access through butler gen3 represents an important change in the interface and operation of the MTAOS.
+The following is a series of improvements that can be implemented to improve the MTAOS usability.
 
-.. Please do not modify tocdepth; will be fixed when a new Sphinx theme is shipped.
+Processing calibration products
+-------------------------------
 
-.. sectnum::
+This is currently done via the ``processCalibrationProducts`` command, which expected a directory containing the raw calibration data.
+Instead, the butler instances hosting the data will also contain processed calibration products suitable for all exposures.
+This will have to be carefully managed by observatory personnel over time, but is a much more suitable approach than leaving the MTAOS in charge of the process.
```

Comment:
We may need to make sure this will fulfill the 2 days requirement defined in 3.6.2 in LTS-186 v2.

Reply:
Good point. In any case, I think this requirement will have to be fulfilled irrespective of this change.